### ISSUE TITLE
Add `attrs` as an explicit package dependency

### ISSUE THREAD

**Issue by joshuadavidthomas** [nicoddemus/pytest-rich#52]
I understand that through the `pytest` dep we get `attrs` along with it, but since the package explicitly imports and relies on it, `attrs` should be included as a direct dependency of this package.

**Comment by darlingm** [comment]
Fixed by 14a0c3f.

**Comment by nicoddemus** [comment]
Thanks @darlingm for the ping. :+1:

**Comment by larsblumberg** [comment]
Would be great to have this released to pypi :-)

**Comment by nicoddemus** [comment]
Good point, I will prepare a release later. 👍

**Comment by nicoddemus** [comment]
Done a `0.2.0` release, which was long overdue. 👍

**Comment by larsblumberg** [comment]
Thanks @nicoddemus.

I have to admit that I removed `pytest-rich` as a dependency b/c I could not get it to work with

```ssh
pytest --exitfirst --looponfail -vv
```

which I used for TDD as it continuously reruns all tests and stops on the first failing test. Especially for TDD I wanted to take profit from seeing all variable values.

`pytest-rich`  however does work with

```
pytest --rich -vv
```

which I use for automated CI/CD.
